runit-nodejs/README.md

```diff
@@ -1,56 +0,0 @@
-socrata/runit
```

Comment:
This info is in the `runit` container's README. This file is pure copypasta and doesn't say anything about `runit-nodejs` specifically.